.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,6 +2,6 @@
 
  - [ ] Closes #xxxx
  - [ ] Tests added
- - [ ] Ranned `isort .` at top--level
+ - [ ] Ranned `isort .` and `black .` at top--level
```

Comment:
Do you mean "ran"?

Reply:
yes! thanks!

Reply:
I'd also change "reproducable" found in the bugs template. Perhaps "Code to reproduce the issue" or something similar would be more clear?